modules/ossm-release-2-6-2.adoc

```diff
@@ -0,0 +1,46 @@
+////
+Module included in the following assemblies:
+* service_mesh/v2x/servicemesh-release-notes.adoc
+////
+
+:_mod-docs-content-type: REFERENCE
+[id="ossm-release-2-6-2_{context}"]
+= {SMProductName} version 2.6.2
+
+This release of {SMProductName} updates the {SMProductName} Operator version to 2.6.2, and includes the following ServiceMeshControlPlane resource version updates: 2.6.2, 2.5.5 and 2.4.11.
+
+This release addresses Common Vulnerabilities and Exposures (CVEs); contains bug fixes; and is supported on {product-title} 4.14 and later.
+
+include::snippets/ossm-current-version-support-snippet.adoc[]
+
+The most current version of the {KialiProduct} can be used with all supported versions of {SMProductName}. The version of {SMProductShortName} is specified by using the `ServiceMeshControlPlane` resource. The version of {SMProductShortName} automatically ensures a compatible version of Kiali.
+
+[id=ossm-release-2-6-2-components_{context}]
+== Component updates
+
+|===
+|Component |Version
+
+|Istio
+|1.20.8
+
+|Envoy Proxy
+|1.28.7
+
+|Kiali Server
+|1.73.15
+|===
+
+[id="ossm-new-feature-2-6-2_{context}"]
+== New feature
+
+* The cert-manager tool is now supported on {ibm-power-title}, {ibm-z-title}, and {ibm-linuxone-name}.
```

Comment:
Is the "cert-manager tool" = "cert-manager Operator for Red Hat OpenShift"? If so, there is an attribute for it:
```suggestion
* The {cert-manager-operator} is now supported on {ibm-power-title}, {ibm-z-title}, and {ibm-linuxone-name}.
```